utils/compress.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python3
+from datetime import datetime
+import os
+import sys
+sys.path.append("..")
+import settings
+import re
+import tarfile
+import settings
+import boto3
+import logging
+
+S3_BUCKET_NAME = getattr(settings, "S3_BUCKET_NAME", None)
```

Comment:
So, `boto3` doesn't require this if you have awscli set up.
I'd prefer we do it that way.
http://boto3.readthedocs.io/en/latest/guide/configuration.html